src/tachyons-base.css

```diff
@@ -31,3 +31,14 @@ a {
 code {
   color: #59cd90; /* green */
 }
+
+
+/* Max-width */
+.eh-mw9 {
+  max-width: 76rem;
+}
+
+/* Transitions */
+.eh-transition {
+  transition: all 150ms ease-in-out;
+}
```

Comment:
Seems that this should go with the other animation related classes:
![image](https://cloud.githubusercontent.com/assets/5497885/25147396/a050ecae-2434-11e7-92f1-dd9f7a691286.png)